explainer.md

```diff
@@ -42,43 +42,35 @@ VR provides an interesting canvas for artists looking to explore the possibiliti
- 7. Continue producing frames until the user indicates that they wish to exit VR mode.
- 8. End the VR session.
+ 1. Request a VR device that supports the presentation modes the application needs.
+ 1. If a device is available, application advertises VR functionality to the user.
+ 1. User performs an action that indicates they want to enter VR mode.
+ 1. Request a VR session from the device to present VR content with.
+ 1. Begin a render loop that produces graphical frames to be displayed on the VR device.
+ 1. Continue producing frames until the user indicates that they wish to exit VR mode.
+ 1. End the VR session.
 
-### Device enumeration
+### Acquiring a Device
 
-The first thing that any VR-enabled page will want to do is enumerate the available VR hardware and, if present, advertise VR functionality to the user.
+The first thing that any VR-enabled page will want to do is request a `VRDevice` and, if present, advertise VR functionality to the user.
 
-`navigator.vr.getDevices` returns a [`Promise`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) that resolves to a list of available devices. Each `VRDevice` represents a physical unit of VR hardware that can present imagery to the user somehow, referred to here as a "VR hardware device". On desktop clients this will usually be a headset peripheral; on mobile clients it may represent the mobile device itself in conjunction with a viewer harness (e.g., Google Cardboard or Samsung Gear VR). It may also represent devices without stereo presentation capabilities but more advanced tracking, such as Tango devices.
+`navigator.vr.requestDevice` returns a [`Promise`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) that resolves to an available device which matches the given filter criteria. A `VRDevice` represents a physical unit of VR hardware that can present imagery to the user somehow, referred to here as a "VR hardware device". On desktop clients this will usually be a headset peripheral; on mobile clients it may represent the mobile device itself in conjunction with a viewer harness (e.g., Google Cardboard or Samsung Gear VR). It may also represent devices without stereo presentation capabilities but more advanced tracking, such as Tango devices.
 
 ```js
 let vrDevice = null;
 
-navigator.vr.getDevices().then(devices => {
-  if (devices.length > 0) {
-    // Use the first device in the array if one is available. If multiple
-    // devices are present, you may want to provide the user a way of choosing
-    // which device to use.
-    vrDevice = devices[0];
-    onVRAvailable();
-  } else {
-    // Could not find any VR hardware connected.
-  }
-}, err => {
-  // An error occurred querying VR hardware. May be the result of blocked
-  // permissions by a parent frame.
+navigator.vr.requestDevice().then((device) => {
+  vrDevice = device;
+  onVRAvailable();
```

Comment:
do you think we might as well have this example pass `vrDevice` as a parameter to the [`onVRAvailable` function](https://github.com/w3c/webvr/blob/8cd20faf/explainer.md#detecting-and-advertising-vr-mode)? as "best practices" sample code, having the `onVRAvailable` example function accept a single argument could be nicer than exposing a global variable.

Reply:
Done, though I still set the global variable in onVRAvailable simply because it's easier to track over the various example snippets than constantly passing the device around.